manifests/barbican-kms/pod.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1 
+kind: Pod 

```

Comment:
We could make this a DS to run on all master nodes. Similar to [0] and [1].
[0] https://github.com/kubernetes/cloud-provider-openstack/blob/master/manifests/controller-manager/openstack-cloud-controller-manager-ds.yaml#L26
[1] https://github.com/kubernetes/website/blob/1d1ccd974d4a6910cd24877ea96b24391ce3a7d1/content/en/examples/admin/cloud/ccm-example.yaml#L57

Reply:
@strigazi we cannot run this as kubernetes construct, It has to be run as docker conatiner or systemd services, I will explore to run it as static pod later

Reply:
When you add trust support I can start testing in our setup.

Static pods works perfectly in a single master cluster. We can see for multimaster.

Reply:
> @strigazi we cannot run this as kubernetes construct, It has to be run as docker conatiner or systemd services, I will explore to run it as static pod later

I think you are talking about different deployment models. I guess @adisky 's original plan is to deploy the kms plugin together with k8s master before the master provision, k8s cannot function well without the kms plugin because it needs to store/retrieve things from etcd. But @strigazi 's meaning is we can deploy kms plugin after k8s installation is completed with no etc encryption and config the kms plugin afterwards.

However, i think static pod can do things pretty well, the only concern is that static pod may be deprecated in the future according to k8s official doc.